content/posts/02_staff_expectations/index.md

```diff
@@ -0,0 +1,116 @@
+* Help the team with project management and prioritization.
+* Identify hidden risks and dependencies, which could disturb project delivery.
+* Advocate with leadership and find the necessary resource to finish a project.
+* Find a way to solve a particularly difficult problem.
+
+## Be a role model and mentor others
+![](./arrows.png)
+
+At some point in a career, you usually start to have a bigger impact through others, rather than by your individual contributions. This 
+happens not only on the management track but also for technical leaders. Some foretaste comes even for senior engineers, but you get a full
+experience of it in staff+ positions. 
+
+Younger people start to look up to staff engineers and even try to mimic their behavior. Staff engineers become role models for
+others. That's why it's important to demonstrate the company's values and culture through actions. They get the leverage to align people and 
+multiply the outcome.
+
+Staff engineers also get an opportunity (and obligation) to 
+[mentor](https://blog.pragmaticengineer.com/developers-mentoring-other-developers/) other people. We consider it a great privilege, to be 
+able to pass your experiences and your knowledge to other people. Helping people grow and succeed is very important. Not only you're 
+actively using your time for others, but also you're increasing the area of impact. We also believe, that during mentoring, you can learn 
+a tremendous amount of things about yourself, efficient communication, setting goals, etc.
+
+Examples, of how to influence others:
+* Participate in or start a mentoring program at your company.
+* Actively demonstrate company values and culture through your actions
+* Review code, documents, proposals, etc.
+* Give feedback!
+
+## Communicate clearly and bring clarity
+![](./communication.png)
+
+Miscommunication is often a source of most problems at work (and in life). Misunderstanding can result in poor performance, subpar
+quality, stressed people, or wasted time. The role of staff engineers is to bring 
+[clarity and alignment](https://alexewerlof.substack.com/p/6-ways-staffs-reach-clarity).
+
+There are several things staff engineers should do to improve communication in their teams and company:
+* Listen carefully to others and understand their problems and point of view.
+* Make sure the team reaches a collective understanding.
+* [Start with why](https://simonsinek.com/books/start-with-why/) and explain why we work on certain things.
+* Lead by example and improve their communication skills. Show more junior engineers this is often a key to success.
+
+## Bring calm
+![](./calm.jpg)
+
+There are going to be some stressful situations at work. It can be either a tight deadline or an application incident. Staff engineers should
+show their experience and stay calm during the crisis. They should take a lead, communicate clearly and help with prioritization. Their role
+is to help team members do their work better.
```

Comment:
The vague statement `Their role is to help team members do their work better.` I don't think it brings anything to the table.

Reply:
Changed a bit.